aten/src/ATen/native/native_functions.yaml

```diff
@@ -11240,3 +11241,6 @@
 - func: unflatten_dense_tensors(Tensor flat, Tensor[] tensors) -> Tensor[]
   variants: function
   python_module: nn
+
+- func: _nested_tensor(Tensor[] list, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
Still no kwarg only after the list? Well it doesn't matter too much here...

BTW, in the minimal formulation, it might be helpful to also have a constructor that takes the compressed Tensor representation directly, instead of having to copy things in. But @swolchok can always add that if he needs it, I suppose.

Reply:
OK I filed an issue for this at https://github.com/pytorch/pytorch/issues/73505

However, to me, this is a pretty good indication that you should have exposed the buffer+nested size constructor (with no TensorOptions arguments) rather than this constructor. You can then provide this implementation in Python.